spec/models/manageiq/providers/redhat/infra_manager/refresher/refresh_recording_modifier_spec.rb

```diff
@@ -7,10 +7,9 @@
   before(:each) do
     init_defaults
     init_connection_vcr
+    stub_const('ORIG_YML_PATH', 'spec/vcr_cassettes/manageiq/providers/redhat/infra_manager/refresh/ovirt_sdk_refresh_recording_for_mod.yml'.freeze)
   end
 
-  ORIG_YML_PATH = 'spec/vcr_cassettes/manageiq/providers/redhat/infra_manager/refresh/ovirt_sdk_refresh_recording_for_mod.yml'.freeze
```

Comment:
Does this even need to be a constant?

Reply:
Maybe you're right. I thought it was something hooked into, but now that I look closer I think it's just being used as a global.